src/tests/Microsoft.PowerFx.Interpreter.Tests/PowerFxEvaluationTests.cs

```diff
@@ -91,7 +91,7 @@ protected override async Task<RunResult> RunAsyncInternal(string expr, string se
 
                 if (iSetup.HandlerName != null)
                 {
-                    if (!SetupHandlers.TryGetValue(setupHandlerName, out var handler))
+                    if (!SetupHandlers.TryGetValue(iSetup.HandlerName, out var handler))
```

Comment:
>iSetup [](http://example.com/codeflow?start=51&length=6)

Why do we need this now?

Reply:
This was suggested by @CarlosFigueiraMSFT. If I understood correctly, in the event where we have more than one setup handler in a text file (e.g. #SETUP: EnableExpressionChaining, AnotherHandler), the setupHandlerName will be a concat value of all handlers.
@CarlosFigueiraMSFT please correct me if this is not the case.

Reply:
This was the case when we needed to have two handlers, but in this PR this is not necessary. We can revert this change now, and add it when we do need it.